company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/README.md

```diff
@@ -2,50 +2,20 @@
 description: 50%
 ---
 
-# How to on-board a vendor
+# Vendor Onboarding
 
-**Effective Date**: January 1, 2020
+**Effective Date**: January 1, 2021
 
-If you are a new full-time staff member who will be issuing Mattermost a monthly or bi-monthly invoice, new vendor supplying goods or services, or a contractor to Mattermost, you will need to go to on-board as a new vendor with the [Finance Team](https://community.mattermost.com/private-core/channels/expenses). 
+If you're a new full-time staff member who will be issuing Mattermost a monthly or bi-monthly invoice, new vendor supplying goods or services, or a contractor to Mattermost, you will need to go to onboard as a new vendor with the [Finance Team](https://community.mattermost.com/private-core/channels/expenses).
 
 ## New Vendor or Contractor
 
-After your contract has gone through our [procurement process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract), the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) will set up your vendor in [Bill.com](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid#payments-made-by-mattermost). For your vendor's profile to be created successfully, please complete the following:
+After your contract has gone through our [procurement process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract), the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) will set up your vendor in Accrualify. The Finance team will send the vendor a Welcome email to the vendor portal. In the portal, the vendor can provide their basic information and banking information, as well as provide their appropriate tax documents like:
```

Comment:
This section is already outdated and needs to be fixed again. 

This section should read: 

"After your contract has gone through our [Procurement Process](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract), the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) will set up your vendor profile in Accrualify.
 Please complete the following forms:

1. [New Vendor Information Form ](https://forms.gle/D27fwJPMXvftVyAEA)
2. [Banking Form](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid#banking-form) under the [How to get paid](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid#payments-made-by-mattermost) section of the Mattermost Handbook
3. Appropriate Tax Form as it applies to you:
      1. [Form W-9](https://www.irs.gov/pub/irs-pdf/fw9.pdf) \(please click [here](https://www.irs.gov/pub/irs-pdf/iw9.pdf) for IRS guidelines\) (US vendors)
     2. [Form W-8BEN](https://www.irs.gov/pub/irs-pdf/fw8ben.pdf) \(please click [here](https://www.irs.gov/pub/irs-pdf/iw8ben.pdf) for IRS guidelines\) (Non-US vendors)
     3. [Form W-8BENE](https://www.irs.gov/pub/irs-pdf/fw8bene.pdf) \(please click [here](https://www.irs.gov/pub/irs-pdf/iw8bene.pdf) for IRS guidelines\) (Non-US vendors)"